packages/contracts-bedrock/contracts/test/OptimismPortal.t.sol

```diff
@@ -318,43 +505,33 @@ contract OptimismPortal_Test is Portal_Initializer {
         uint8 _gasLimit,
         bytes memory _data
     ) external {
-        // Cannot call the optimism portal
-        vm.assume(_target != address(op));
         uint256 _nonce = messagePasser.nonce();
-
+        Types.WithdrawalTransaction memory _tx = Types.WithdrawalTransaction({
```

Comment:
This test is annoying because of stack too deep errors, it looks like using the struct here helps

Reply:
Makes the code cleaner IMO too.